mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMapperProvider.java

```diff
@@ -129,4 +138,9 @@ public AnnotationMapper createAnnotationMapper() throws MailboxException {
     public List<Capabilities> getNotImplemented() {
         return ImmutableList.of();
     }
+
+    @Override
+    public MessageUid generateMessageUid() {
+        return MessageUid.of(RandomUtils.nextLong());
```

Comment:
That's not correct, Uids should be monotically incrising per mailbox.

Maybe we should write a wrapper around a Int we increase on each call to next, that would be more correct...